pegasus/sites.v3/hourofcode.com/views/home_signup.haml

```diff
@@ -3,7 +3,9 @@
 #join
   %h1#join-us-header=hoc_s(:front_join_us_button)
   -if ["post-hoc", false].include?(hoc_mode)
-    %h3#signup-closed=hoc_s(:signup_closed)
+    #signup-closed
+      =hoc_s(:signup_registration_not_required).gsub(/%{hoc_activities_url}/, resolve_url('/learn')).gsub(/%{howto_guide_url}/, resolve_url('/how-to'))
```

Comment:
I bet we could write a little helper here that just took an array of hashes for substitutions.

Reply:
(For later.)